transports/transport-rest/transport-rest-vertx/src/test/java/org/apache/servicecomb/transport/rest/vertx/TestRestServerVerticle.java

```diff
@@ -109,6 +109,38 @@ public void testRestServerVerticleWithRouterSSL(@Mocked Transport transport, @Mo
     server.start(startFuture);
   }
 
+  @Test
+  public void testRestServerVerticleWithHttp2(@Mocked Transport transport, @Mocked Vertx vertx,
+      @Mocked Context context,
+      @Mocked JsonObject jsonObject, @Mocked Future<Void> startFuture) throws Exception {
+    URIEndpointObject endpointObject = new URIEndpointObject("http://127.0.0.1:8080?protocol=h2");
+    new Expectations() {
+      {
+        transport.parseAddress("http://127.0.0.1:8080?protocol=h2");
```

Comment:
ALPN(h2) cannot be used for http schema, only https is allowed to use h2 protocol.